drivers/axi_core/spi_engine/spi_engine.c

```diff
@@ -140,6 +140,23 @@ static int32_t spi_engine_set_transfer_width(struct spi_desc *desc,
 	return SUCCESS;
 }
 
+/**
+ * @brief Set SPI engine clock frequency
+ *
+ * @param desc Decriptor containing SPI Engine's parameters
+ * @param speed_hz SPI engine transfer speed
+ */
+void spi_engine_set_speed(struct spi_desc *desc,
+			  uint32_t speed_hz)
+{
+	struct spi_engine_desc	*desc_extra;
+
+	desc_extra = desc->extra;
+
+	desc_extra->clk_div =  desc_extra->ref_clk_hz /
+			       (2 * speed_hz) - 1;
```

Comment:
I see this method of calculating the divide is used in at least two places. Maybe it could be a define or inline function to avoid changing one and not the other in the future.

Reply:
Was this change ignored or missed ? The argument was valid in my opinion